mtp_cashbook/assets-src/javascripts/app.js

```diff
@@ -17,5 +17,72 @@
   checks.on('change', function() {
     var check = $(this);
     check.closest('tr')[check.is(':checked') ? 'addClass' : 'removeClass']('is-selected');
-  })
+  });
+
+  var tourBase = {
```

Comment:
I'm not sure that really fits into the module pattern of Heisenburg. It's simply settings in an object literal for an external component.